doc/Requirements Specification/Parts/Non-Functional Requirements.lyx

```diff
@@ -858,7 +870,20 @@ To analyse the entire project, there is an entry
 \begin_inset Quotes eld
 \end_inset
 
-Analyse with Beagle
+Analyse with 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+gls{Beagle}
+\end_layout
```

Comment:
I just wanted to make sure no one has good argument for or against it before we use one of the options. I think it doesn't really matter.